Solver_interface/include/CGAL/Eigen_matrix.h

```diff
@@ -180,6 +180,56 @@ struct Eigen_sparse_matrix
       m_triplets.push_back(Triplet(i,j,val));
   }
 
+  /// Read access to a matrix coefficient.
+  ///
+  /// Complexity:
+  /// - O(log(n)) if the matrix is already built.
+  /// - O(n) if the matrix is not built.
+  /// n being the number of entries in the matrix.
+  ///
+  /// @commentheading Preconditions:
+  /// - 0 <= i < row_dimension().
+  /// - 0 <= j < column_dimension().
+  NT get_coef (unsigned int i, unsigned int j) const
+  {
+    CGAL_precondition(i < row_dimension());
+    CGAL_precondition(j < column_dimension());
+
+    NT val;
+    if (m_is_already_built)
+        val = m_matrix.coeffRef(i,j);
+    else
+    {
+        // with lambda
+        auto pred = [i, j](const Triplet& triplet)
+        {
+            return triplet.row() == i && triplet.col() == j;
+        };
+        typename std::vector<Triplet>::iterator it;
+        it = std::find_if(m_triplets.begin(), m_triplets.end(), pred);
+        Triplet tr = *it;
```

Comment:
This will not work if the triple is not here. In such a case you should return 0. By the way a simple for-loop might be sufficient here (as c++11 is not allowed in this central package)